src/algorithm/geometry.hxx

```diff
@@ -173,14 +174,67 @@ namespace se3
       radius = std::max (jMb.act(SE3_GEOM_AABB(fcl,max,max,max)).squaredNorm(),radius);
 
       // Don't forget to sqroot the squared norm before storing it.
-      assert (geom.parent<geomData.radius.size());
-      geomData.radius[geom.parent] = sqrt(radius);
+      geomData.radius[i] = sqrt(radius);
     }
   }
 
 #undef SE3_GEOM_AABB
 #endif // WITH_HPP_FCL
 
+  inline void appendGeometryModel(GeometryModel & geomModel1,
+                                  const GeometryModel & geomModel2)
+  {
+    assert (geomModel1.ngeoms == geomModel1.geometryObjects.size());
+    Index nGeom1 = geomModel1.geometryObjects.size();
+    Index nColPairs1 = geomModel1.collisionPairs.size();
+    assert (geomModel2.ngeoms == geomModel2.geometryObjects.size());
+    Index nGeom2 = geomModel2.geometryObjects.size();
+    Index nColPairs2 = geomModel2.collisionPairs.size();
+
+    /// Append the geometry objects and geometry positions
+    geomModel1.geometryObjects.insert(geomModel1.geometryObjects.end(),
+        geomModel2.geometryObjects.begin(), geomModel2.geometryObjects.end());
+    geomModel1.ngeoms += nGeom2;
+
```

Comment:
Actually, I realize now I should not be talking about radius here. Radius is an attribute of `GeometryData`. When modifing `GeometryModel`, users should understand that the corresponding `GeometryData` is not valid anymore, shouldn't they ?

I would rather not add a `GeometryData` as argument.